app/views/recipes/show.html.erb

```diff
@@ -0,0 +1,71 @@
+<%= render partial: "shared/add_ingredient" %>
+
+<% if RecipeFood.find_by(id: params[:ingredient_id])%>
+  <%= render partial: "shared/edit_ingredient" %>
+<% end %>
+
+<p style="color: green"><%= notice %></p>
+
+<div class="card">
+  <div class="card-body d-flex">
+    <div class="title w-100">
+      <div class="d-flex justify-content-between">
+        <h4><%= @recipe.name %></h4>
+        <% if can? :manage, @recipe %>
+          <div class="d-flex">
+            <%= button_to "Delete Recipe", recipe_path(@recipe.id), method: :delete, class:"text-white btn btn-danger px-1 py-0 mx-1" %>
+            <%= button_to "Edit Recipe", edit_recipe_path(@recipe.id), method: :get,class:"text-white btn btn-secondary px-1 py-0" %>
+          </div>
+        <% end %>
+      </div>
+      <div class="d-flex justify-content-between">
+        <p>Preparation time: <%= @recipe.preparation_time %></p>
+        <p>public: <%= @recipe.public %></p>
+      </div>
+      <p>cooking time: <%= @recipe.cooking_time %></p>
+      <p><%= @recipe.decription %></p>
+    </div>
+    <div class="details"></div>
+  </div>
+</div>
+
+<div class="d-flex justify-content-between my-4">
+  <%= link_to "Generate shopping list", "/general_shopping_list/#{@recipe.id}", class: "btn btn-info" %>
+  <button class="btn btn-success" id="addIngredient" data-toggle="modal" data-target="#exampleModal" onclick="(() => {
+    document.getElementById('popupAdd').classList.toggle('d-none')
+  })()">Add ingredient</button>
```

Comment:
There is an error when I try to add ingredients to the recipe on the recipe show page. Kindly fix the error.
![ingredient_error](https://user-images.githubusercontent.com/35504149/169378665-9db2d042-c6ba-4259-ae58-295b8d575ace.png)